src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -746,12 +748,19 @@ internal async Task SendInvocationRequest(ScriptInvocationContext context)
                     return;
                 }
 
-                // do not send an invocation request if cancellation has been requested
                 if (context.CancellationToken.IsCancellationRequested)
                 {
-                    _workerChannelLogger.LogWarning("Cancellation has been requested. The invocation request with id '{invocationId}' is canceled and will not be sent to the worker.", invocationId);
-                    context.ResultSource.TrySetCanceled();
-                    return;
+                    _workerChannelLogger.LogInformation("Cancellation was requested prior to the invocation request ('{invocationId}') being sent to the worker. " +
+                                                        "For more information see https://aka.ms/AAmnbwj", invocationId);
```

Comment:
is this the right aka link? can we make it more readable? (azure-functions-cancellations, or something similar?)

Reply:
Can give it a proper display name; currently pointing to just the cancellation docs but will update the link in aka.ms to point to the new docs once released

Reply:
FWIW, @mattchenderson created few starts with "https://aka.ms/azure-functions/" . So might consider using the same format:
> https://aka.ms/azure-functions/cancellations ?